### PR TITLE
Fix graphs and search #1, #2

### PR DESCRIPTION
**Pull request by wrooot**

### CODE REVIEW

**Comment by wrooot** [comment]
This is fairly critical as people will start running into problems after upgrading to 4.3.0. This change was proposed by a user in GitHub and chat (i haven't tested though). Not sure if i have done the commits correctly. @guusdk said i should do snapshot incrementing in the same PR.

**Comment by wrooot** [comment]
@suf4dev did similar PR https://github.com/igniterealtime/openfire-monitoring-plugin/pull/22 but mine has updated version and changelog.

File: pom.xml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.3.0</modelVersion>
```
Comment (guusdk) [inline pom.xml]: It is an error to change the modelVersion value. It is a value that is specific to Maven, not Openfire. Please keep this at 4.0.0.

Comment (wrooot) [reply pom.xml]: I think i have fixed it

**Comment by guusdk** [comment]
Thanks!

**Pull request merged**